.circleci/config.yml

```diff
@@ -0,0 +1,136 @@
+version: 2.1
+
+
+commands:
+  npm-install:
+    steps:
+      - run: sudo npm install npm@6 --global
+      - restore_cache:
+          key: js-deps-{{ checksum "package.json" }}
+      - run: npm install --no-save
+      - save_cache:
+          key: js-deps-{{ checksum "package.json" }}
+          paths:
+            - ./node_modules
+
+  pip-install-docs:
+    steps:
+      - run: sudo pip install --upgrade pip
```

Comment:
Is this really needed? What's wrong with the pip version installed in the docker image `circleci/python:3.6`?